storehaus-algebra/src/test/scala/com/twitter/storehaus/algebra/query/CalendarTimeQueryRangeProperties.scala

```diff
@@ -19,7 +19,8 @@ package com.twitter.storehaus.algebra.query
 import org.scalacheck.{ Gen, Arbitrary }
 import org.scalacheck.Prop._
 import org.scalacheck.Properties
-import com.twitter.scalding._
+import com.twitter.scalding.{RichDate, DateRange, Duration, AbsoluteDuration, Years,
```

Comment:
+1 to minimize. I prefer to keep the diffs as minimal as possible (and personally never mix style and logic changes if possible).